README.md

```diff
@@ -70,7 +70,7 @@ To support tight fitting volumes for a variety of datasets from regularly divide
 
 _TODO: include a screenshot of each bounding volume type._
 
-A tile usually references a _model_ or set of _models_, e.g., 3D buildings.  These models may be batched together into essentially a single model to reduce client-side load time and WebGL draw call overhead.
+A tile references a _feature_ or set of _features_, such as 3D models representing buildings or trees, points in a point cloud, or polygons, polylines, and points in a vector dataset.  These features may be batched together into essentially a single feature to reduce client-side load time and WebGL draw call overhead.
```

Comment:
Just an FYI, in geospatial terms, multiple features (heterogeneous or not) are almost always referred to as a `feature collection`.